### PR TITLE
My profile refinement

### PR DESCRIPTION
**Pull request by maciaszczykm**
## Summary
Related to [ENG-307](https://linear.app/pluralsh/issue/ENG-307/my-profile-refinement).
Closes ENG-426.

[Figma design](https://www.figma.com/file/Y8iNQX9MTG2uZIjNelBroF/My-Profile?node-id=245%3A14643).

Used packages-redesign as a base branch here but I can switch to some other branch if you would like to.

Zrzut ekranu 2022-07-29 o 14 06 39
Zrzut ekranu 2022-07-29 o 14 06 45
Zrzut ekranu 2022-07-29 o 14 07 09

## Test Plan
Tested manually.

## Checklist
- [x] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [x] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

File: www/src/components/utils/Header.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { Box } from 'grommet'
+import { Flex, H1, Span } from 'honorable'
+
+export function Header({ header, description, children }) {
```
Comment (floreks) [inline www/src/components/utils/Header.js:4]: Is it used? I didn't find any usage in the new code.

Comment (maciaszczykm) [reply www/src/components/utils/Header.js:4]: It's imported in `plural/www/src/components/account/AccountAttributes.js` and `plural/www/src/components/repository/OIDCProvider.js`.

**Review by dogmar — CHANGES_REQUESTED** [review]
Need to add [profile picture switch/delete functionality](https://www.figma.com/file/Y8iNQX9MTG2uZIjNelBroF/My-Profile?node-id=247%3A14484). 

Bottom of profile picture doesn't align with Upload button. I think if you switch out the Avatar component for the new IconFrame from the design system, that will fix it. 
Screen Shot 2022-07-29 at 8 59 47 AM

**Comment by dogmar** [comment]
On the security screen, if you enter a valid password, then select all of the text in the password field and delete it at once, the hint still says "Valid password" even though the field is empty.

**Comment by maciaszczykm** [comment]
> Bottom of profile picture doesn't align with Upload button. I think if you switch out the Avatar component for the new IconFrame from the design system, that will fix it.

Nice catch! `Stack` was missing height property and that was causing the issue. I've tried using `IconFrame` but it has some issues with aspect ratios. When image is not a square, then it forces it and it looks worse than when using `Avatar`.

Zrzut ekranu 2022-08-1 o 11 48 01

Also, I think we cannot replace `Avatar` with `IconFrame` as logic for creating default picture is part of the first one.

**Comment by maciaszczykm** [comment]
> On the security screen, if you enter a valid password, then select all of the text in the password field and delete it at once, the hint still says "Valid password" even though the field is empty.

Nice catch. I didn't change that part yet as I need some input from design team but anyways it doesn't work as expected because `validation` method is not called when doing `CTRL` + `A` and then `DELETE`. I've opened [a new bug to track this](https://linear.app/pluralsh/issue/ENG-439/validation-is-not-called-when-doing-ctrl-a-and-then-delete-in).

**Comment by maciaszczykm** [comment]
> Need to add [profile picture switch/delete functionality](https://www.figma.com/file/Y8iNQX9MTG2uZIjNelBroF/My-Profile?node-id=247%3A14484).

I wanted to add this from separate PR/[subissue](https://linear.app/pluralsh/issue/ENG-426/add-a-way-to-deletereset-profile-picture-to-default) but there is no need to split as it's small change. Already done.

@dogmar Thanks for precise review, PTAL.

**Review by michaeljguarino — APPROVED** [review]

**Comment by michaeljguarino** [comment]
@dogmar could you give this a second look, I believe your review is the only thing blocking

File: www/src/components/utils/Header.js (hunk 1 above)
Comment (floreks) [reply www/src/components/utils/Header.js:4]: It's imported from `src/components/profile/Header.js` there, not this one from what I can see.

**Comment by floreks** [comment]
We'd probably need to add a custom upload modal with crop functionality to be able to use icon frame or check if the picture has the proper ratio before upload.

File: www/src/components/utils/Header.js (hunk 1 above)
Comment (maciaszczykm) [reply www/src/components/utils/Header.js:4]: https://github.com/pluralsh/plural/blob/9b66cc61305f31398bb324871bc68c5acdd72cd0/www/src/components/account/AccountAttributes.js#L10-L11

Comment (maciaszczykm) [reply www/src/components/utils/Header.js:4]: You're right, good catch. I've started this to move Header to utils as it's used not only in the profile but across whole app. Probably switched to something and forgot to finish.

**Review by dogmar — APPROVED** [review]
Thanks for addressing all my comments. Looks good!

**Pull request merged**